.github/dependabot.yml

```
@@ -0,0 +1,21 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  - package-ecosystem: "npm" # See documentation for possible values
+    directory: "/src/client-admin-portal" # Location of package manifests
+    schedule:
+      interval: "weekly"
+
+  - package-ecosystem: "npm" # See documentation for possible values
+    directory: "/src/client-contractor-portal"
+    schedule:
+      interval: "weekly"
+      
+  - package-ecosystem: "pip"
```

I think this should say `uv` instead of `pip` https://docs.github.com/en/code-security/dependabot/ecosystems-supported-by-dependabot/supported-ecosystems-and-repositories
```suggestion
  - package-ecosystem: "uv"
```

---

You're right. The page referring to the setting up the .yaml is not up to date.